resources/locale.json

```diff
@@ -745,195 +745,195 @@
         "UI_PEACOCK_ROCCO": "Der McVeigh Aufstieg",
         "UI_PEACOCK_ROCCO1": "Der McVeigh Aufstieg - Stufe 1",
         "UI_PEACOCK_ROCCO2": "Der McVeigh Aufstieg - Stufe 2",
         "UI_PEACOCK_ROCCO3": "Der McVeigh Aufstieg - Stufe 3",
-        "UI_PEACOCK_ROCCO_OBJ1_TITLE": "Eliminate Rocco",
-        "UI_PEACOCK_ROCCO_OBJ1_DESC": "Eliminate Rocco as a Chef",
-        "UI_PEACOCK_ROCCO_OBJ2_TITLE": "Eliminate Rocco's sister",
-        "UI_PEACOCK_ROCCO_OBJ2_DESC": "Eliminate Rocco's sister as a Chef",
+        "UI_PEACOCK_ROCCO_OBJ1_TITLE": "Eliminiere Rocco",
+        "UI_PEACOCK_ROCCO_OBJ1_DESC": "Eliminiere Rocco als Koch",
+        "UI_PEACOCK_ROCCO_OBJ2_TITLE": "Eliminiere Roccos Schwester",
+        "UI_PEACOCK_ROCCO_OBJ2_DESC": "Eliminiere Roccos Schwester als Koch",
         "UI_PEACOCK_SHANGRILA": "Der Mendietinha Wahnsinn",
         "UI_PEACOCK_SHANGRILA1": "Der Mendietinha Wahnsinn - Stufe 1",
         "UI_PEACOCK_SHANGRILA2": "Der Mendietinha Wahnsinn - Stufe 2",
         "UI_PEACOCK_SHANGRILA3": "Der Mendietinha Wahnsinn - Stufe 3",
-        "UI_PEACOCK_SHANGRILA_BIKES": "Destroy 4 Bikes",
-        "UI_PEACOCK_SHANGRILA_BIKES_DESC": "Let's annoy those pesky bikers.",
-        "UI_PEACOCK_SHANGRILA_SNIPER": "Retrieve the sniper, don't drop it",
-        "UI_PEACOCK_SHANGRILA_SNIPER_DESC": "Retrieve the Bartoli Woodsman Hunting Rifle, and don't drop it.",
+        "UI_PEACOCK_SHANGRILA_BIKES": "Zerstöre 4 Motorräder",
+        "UI_PEACOCK_SHANGRILA_BIKES_DESC": "Lasst uns diese störenden Biker nerven.",
+        "UI_PEACOCK_SHANGRILA_SNIPER": "Besorge das Scharfschützengewehr, verliere es nicht",
+        "UI_PEACOCK_SHANGRILA_SNIPER_DESC": "Besorge das Bartoli Scharfschützengewehr und verliere es nicht.",
         "UI_PEACOCK_REMAINING": "Ziele verbleidend",
         "UI_PEACOCK_PIRATESPROBLEM": "Das Piraten Problem",
         "UI_PEACOCK_PIRATESPROBLEM1": "Das Piraten Problem - Stufe 1",
         "UI_PEACOCK_PIRATESPROBLEM2": "Das Piraten Problem - Stufe 2",
         "UI_PEACOCK_PIRATESPROBLEM3": "Das Piraten Problem - Stufe 3",
-        "UI_PEACOCK_PIRATESPROBLEM_CALM": "Calm the staff",
-        "UI_PEACOCK_PIRATESPROBLEM_CALM_DESC": "Pirates have stolen code from the Haven™ servers, and the staff is in panic.\nCalm the staff by pumping a sedative in their ventilation.",
-        "UI_PEACOCK_PIRATESPROBLEM_ALARM": "Trigger an alarm",
-        "UI_PEACOCK_PIRATESPROBLEM_ALARM_DESC": "Make sure the issue doesn't escalate.",
-        "UI_PEACOCK_PIRATESPROBLEM_PIRATES": "Obtain the list of pirates",
-        "UI_PEACOCK_PIRATESPROBLEM_PIRATES_DESC": "Get the list containing all pirate identities from The Captain.",
+        "UI_PEACOCK_PIRATESPROBLEM_CALM": "Beruhige das Personal",
+        "UI_PEACOCK_PIRATESPROBLEM_CALM_DESC": "Piraten haben Code aus den Haven™ Servern gestohlen und das Personal ist in Aufruhr.\nBeruhige sie indem du Beruhigungsmittel in ihre Lüftung pumpst.",
```

Comment:
I think "Code von den Servern gestohlen" sounds better. "Code aus den Servern gestohlen" sounds like they've taken it out physically.